packages/react/src/switch/root/SwitchRoot.tsx

```diff
@@ -31,7 +31,7 @@ import { useValueChanged } from '../../utils/useValueChanged';
  */
 export const SwitchRoot = React.forwardRef(function SwitchRoot(
   componentProps: SwitchRoot.Props,
-  forwardedRef: React.ForwardedRef<HTMLButtonElement>,
+  forwardedRef: React.ForwardedRef<HTMLSpanElement>,
```

Comment:
`HTMLElement` alone may be better to allow changing to `<button>`, which `<Button>` itself does